rust/gui-client/src-common/Cargo.toml

```
@@ -0,0 +1,37 @@
+[package]
+name = "firezone-gui-client-common"
+# mark:next-gui-version
+version = "1.3.1"
```

```suggestion
version = "0.1.0"
```

Does this need to track the GUI version?

---

I thought it wouldn't hurt 🤷‍♀️